protostar/profiler/profile.py

```diff
@@ -15,20 +18,25 @@
 from starkware.cairo.common.cairo_function_runner import CairoFunctionRunner
 from starkware.cairo.lang.vm.memory_segments import MemorySegmentManager
 from protostar.profiler.pprof import serialize, to_protobuf
+if TYPE_CHECKING:
+    from protostar.starknet.cheatable_execute_entry_point import ContractProfile
 
 Address = int
 StringID = int
 FunctionID = int
 
+import random
+def unique_id():
+    return random.randint(0, 10000000000)
```

Comment:
```suggestion
_unique_id_counter = 0
def unique_id():
    global _unique_id_counter
    _unique_id_counter += 1
    return _unique_id_counter
```

This is more collision proof. Or use UUIDv4 if you need global uniqueness.

Reply:
I will have to change the alghoritm (forgot about it). This solution won't work because it is used as an offset so we need some free spots after each id.

Reply:
#832